Calculator/LinkedListTest/LinkedListTest.swift

```diff
@@ -0,0 +1,144 @@
+//
+//  LinkedListTest.swift
+//  LinkedListTest
+//
+//  Created by 강민수 on 2023/01/26.
+//
+
+import XCTest
+@testable import Calculator
+
+extension String: CalculateItem {
+}
+
+final class LinkedListTest: XCTestCase {
+    
+    var sut: LinkedList<String>!
+
+    override func setUpWithError() throws {
+        try super.setUpWithError()
+        sut = LinkedList()
+    }
+
+    override func tearDownWithError() throws {
+        try super.tearDownWithError()
+        sut = nil
+    }
+    
+    func test_초기_인스턴스_생성시_isEmpty는_true이다() {
+        // then
+        XCTAssertTrue(sut.isEmpty)
+    }
+    
+    func test_a의_값을_append하면_isEmpty는_false이다() {
+        // given
+        let firstValue = "a"
+        // when
+        sut.append(firstValue)
+        // then
+        XCTAssertFalse(sut.isEmpty)
+    }
+    
+    func test_a의_값을_append한다면_head값은_a이다() {
+        // given
+        let value = "a"
+        let exception = "a"
+        // when
+        sut.append(value)
+        let headValue = sut.head!.value
+        // then
+        XCTAssertEqual(headValue, exception)
+    }
+    
+    func test_a의_값을_append한다면_tail값은_a이다() {
+        // given
+        let value = "a"
+        let exception = "a"
+        // when
+        sut.append(value)
+        let tailValue = sut.tail!.value
+        // then
+        XCTAssertEqual(tailValue, exception)
+    }
+    
+    func test_a과_b를_append한다면_head는_a_tail은_b다() {
+        // given
+        let firstValue = "a"
+        let secondValue = "b"
+        let headException = "a"
+        let tailException = "b"
+        // when
+        sut.append(firstValue)
+        sut.append(secondValue)
+        let headValue = sut.head!.value
+        let tailValue = sut.tail!.value
+        // then
+        XCTAssertEqual(headValue, headException)
+        XCTAssertEqual(tailValue, tailException)
+    }
+    
+    func test_a와_b_두개의_Node를_가진_LinkedList에서_removeFirst를_한다면_반환값은_a이다() {
+        // given
+        let firstValue = "a"
+        let secondValue = "b"
+        let exception = "a"
+        // when
+        sut.append(firstValue)
+        sut.append(secondValue)
+        let removedValue = sut.removeFirst()!
```

Comment:
참고로 두 개뿐이긴 하지만 많아진다면 아래와 같은 방식으로 할 수 있습니다~
```suggestion
        // given
        let values = ["a", "b"]
        let exception = "a"
        // when
        values.forEach {
            sut.append($0)
        }
        let removedValue = sut.removeFirst()!
```